modules/37_industry/subsectors/postsolve.gms

```diff
@@ -42,26 +42,45 @@ o37_demFeIndSub(ttot,regi,entySe,entyFe,secInd37,emiMkt)
     * vm_demFeSector_afterTax.l(ttot,regi,entySe,entyFe,"indst",emiMkt)
   );
 
-
 *** FE per subsector whose emissions can be captured (helper parameter for 
 *** calculation of industry captured CO2 below)
-o37_demFeIndSub_SecCC(ttot,regi,secInd37) 
+*** NOTE: not used for pm_IndstCO2Captured anymore since it did not take into
+*** account the different emission factors of FE carriers.  But used in remind2,
+*** so kept for backwards compatibility.
+o37_demFeIndSub_SecCC(ttot,regi,secInd37)
   = sum((se2fe(entySe,entyFe,te),macBaseInd37(entyFe,secInd37),
-                                 sector2emiMkt("indst",emiMkt)), 
+                                 sector2emiMkt("indst",emiMkt)),
       o37_demFeIndSub(ttot,regi,entySe,entyFe,secInd37,emiMkt)
     );
 
-*** industry captured CO2
-pm_IndstCO2Captured(ttot,regi,entySe,entyFe,secInd37,emiMkt)$(
-                        entyFeCC37(entyFe) 
-                        AND o37_demFeIndSub_SecCC(ttot,regi,secInd37)
-                        AND macBaseInd37(entyFe,secInd37)) 
-  = sum( secInd37_2_emiInd37(secInd37,emiInd37)$(emiInd37_fuel(emiInd37)), 
-      vm_emiIndCCS.l(ttot,regi,emiInd37)
-    )
-  * o37_demFeIndSub(ttot,regi,entySe,entyFe,secInd37,emiMkt) 
-  / o37_demFeIndSub_SecCC(ttot,regi,secInd37);
+*** industry captured fuel CO2
+pm_IndstCO2Captured(ttot,regi,entySE,entyFE(entyFEcc37),secInd37,emiMkt)$(
+                  macBaseInd37(entyFE,secInd37)
+              AND sum(se2fe(entySE2,entyFE,te),
+                    o37_demFEindsub(ttot,regi,entySE2,entyFE,secInd37,emiMkt)) )
+  = ( o37_demFEindsub(ttot,regi,entySE,entyFE,secInd37,emiMkt)
+    * sum(se2fe(entySE2,entyFE,te),
+        !! collapse entySE dimension, so emission factors apply to all entyFE
+	!! regardless or origin, and therefore entySEbio and entySEsyn have
+	!! non-zero emission factors
+        pm_emifac(ttot,regi,entySE2,entyFE,te,"co2")
+      )
+    ) !! subsector emissions
 
+  * ( sum(secInd37_2_emiInd37(secInd37,emiInd37(emiInd37_fuel)),
+        vm_emiIndCCS.l(ttot,regi,emiInd37)
+      ) !! subsector captured energy emissions
 
-*** EOF ./modules/37_industry/subsectors/postsolve.gms
+    / sum(se2fe(entySE2,entyFE2,te),
+        o37_demFEindsub(ttot,regi,entySE2,entyFE2,secInd37,emiMkt)
+      * sum(se2fe2(entySE3,entyFE2,te2),
+          !! collapse entySE dimension, so emission factors apply to all entyFE2
+          !! regardless or origin, and therefore entySEbio and entySEsyn have
+          !! non-zero emission factors
+          pm_emifac(ttot,regi,entySE3,entyFE,te2,"co2")
+	)
+      )  !! subsector total emissions
```

Comment:
This should be the same as ``v37_emiIndCCSmax``, no? Could be replaced. Maybe check again that it is the same as the current sum with pm_emifac *  o37_demFEindsub.